src/org/rascalmpl/library/Prelude.java

```diff
@@ -2326,6 +2336,24 @@ public IFunction parser(IValue start,  IBool allowAmbiguity, IBool hasSideEffect
 	public IFunction parsers(IValue start,  IBool allowAmbiguity, IBool hasSideEffects, IBool firstAmbiguity, ISet filters) {
         return rascalValues.parsers(start, allowAmbiguity, hasSideEffects, firstAmbiguity, filters);
     }
+
+	public void storeParsers(IValue start, ISourceLocation saveLocation) {
+		try {
+			rascalValues.storeParsers(start, saveLocation);
+		}
+		catch (IOException e) {
+			throw RuntimeExceptionFactory.io(e.getMessage());
```

Comment:
this can be tested by writing to a non-existing folder or something. however we'd not be testing the actual code that we wrote here, just  the URIResolverRegistry.